### PR TITLE
Hnimrama/atom test readme

### PR DESCRIPTION
**Pull request by hnimra-amd**
# Add ATOM inference suite and config READMEs

**Branch:** `hnimrama/atom-test-readme`  
**Base:** `dev/dtni`

## Summary

Add documentation for the ATOM inference suite, modeled after the JAX MaxText training READMEs:

- New **`cvs/tests/inference/atom/README.md`** — how to run `cvs run atom`, lifecycle, sweeps, metric tiers, quick start
- Updated **`cvs/input/config_file/inference/atom/README.md`** — config/threshold reference (file inventory, must-change table, structure, thresholds, lab workflow)

Docs focus on ATOM workloads only (`driver=atom` single-node, `driver=vllm_atom` multinode PP).

## Test plan

- [x] Docs-only change — review for accuracy against existing variant configs and `atom.py` lifecycle

### CODE REVIEW

**Comment by atnair-amd** [comment]
double check the tests readme and see if it comes up in cvs list. otherwise LGTM

**Comment by hnimra-amd** [comment]
> double check the tests readme and see if it comes up in cvs list. otherwise LGTM

Confirmed, the tests readme doesnt show up in cvs list 

(.cvs_venv) hnimrama@amd.com@tus1-vm-amd-prj2-jumphost-01:~/cvs-multinode$ cvs list

Available Tests
================================================================================

Package: cvs
--------------------------------------------------------------------------------

  cvs.tests.anc (1 test suite)
    • anc_installation

  cvs.tests.anc.cpu (1 test suite)
    • anc_test_cpu

  cvs.tests.anc.gpu (1 test suite)
    • anc_test_gpu

  cvs.tests.benchmark (1 test suite)
    • test_aorta

  cvs.tests.health (4 test suites)
    • agfhc_cvs
    • csp_qual_agfhc
    • rvs_cvs
    • transferbench_cvs

  cvs.tests.health.install (5 test suites)
    • install_agfhc
    • install_babelstream
    • install_rocblas
    • install_rvs
    • install_transferbench

  cvs.tests.ibperf (2 test suites)
    • ib_perf_bw_test
    • install_ibperf_tools

  cvs.tests.inference.atom (1 test suite)
    • atom

  cvs.tests.inference.pytorch_xdit (2 test suites)
    • pytorch_xdit_flux1_dev_single
    • pytorch_xdit_wan22_14b_single

  cvs.tests.inference.sglang (3 test suites)
    • sglang_disagg_distributed
    • sglang_distributed
    • sglang_single

  cvs.tests.inference.vllm (1 test suite)
    • vllm

  cvs.tests.mori (1 test suite)
    • mori_benchmark_test

  cvs.tests.ompi (1 test suite)
    • install_ompi

  cvs.tests.platform (1 test suite)
    • host_configs_cvs

  cvs.tests.preflight (1 test suite)
    • preflight_checks

  cvs.tests.rccl (3 test suites)
    • rccl_pairwise
    • rccl_perf
    • rccl_regression

  cvs.tests.training.jax (3 test suites)
    • jax_llama3_1_405b_distributed
    • jax_llama3_1_70b_distributed
    • jax_llama3_1_70b_single

  cvs.tests.training.megatron (4 test suites)
    • megatron_llama3_1_70b_distributed
    • megatron_llama3_1_70b_single
    • megatron_llama3_1_8b_distributed
    • megatron_llama3_1_8b_single

  cvs.tests.training.torchtitan (8 test suites)
    • torchtitan_deepseek_16b_distributed
    • torchtitan_deepseek_16b_single
    • torchtitan_llama3_1_70b_distributed
    • torchtitan_llama3_1_70b_single
    • torchtitan_llama3_1_8b_distributed
    • torchtitan_llama3_1_8b_single
    • torchtitan_qwen3_32b_distributed
    • torchtitan_qwen3_32b_single

================================================================================
Total: 44 test suites across 1 package(s)
